test/ziggurat/producer_test.clj

```diff
@@ -128,9 +128,9 @@
   (testing "with incorrect config"
     (let [valid-config (assoc valid-config :linger-ms-foo "1")]
       (is (thrown? java.lang.RuntimeException (producer-properties valid-config))))
-    (let [valid-config (update  valid-config :key-serializer-class (constantly  "java.time.Clock"))]
+    (let [valid-config (update valid-config :key-serializer-class (constantly "java.time.Clock"))]
       (is (thrown? java.lang.RuntimeException (producer-properties valid-config))))
-    (let [valid-config (update  valid-config :key-serializer-class (constantly  "java.foo.Bar"))]
+    (let [valid-config (update valid-config :key-serializer-class (constantly "java.foo.Bar"))]
```

Comment:
can this be one let binding?

Reply:
Can you elaborate more? I didn't get it 😅
There's just one variable declared right? `valid-config`

Reply:
can it be different variables using only one let binding?

Reply:
Do you mean something like this?
```
(let [valid-config1 (assoc valid-config :linger-ms-foo "1")
       valid-config2 (update valid-config :key-serializer-class (constantly "java.time.Clock"))
       valid-config3 (update valid-config :key-serializer-class (constantly "java.foo.Bar"))
       ]
      (is (thrown? java.lang.RuntimeException (producer-properties valid-config1)))
      (is (thrown? java.lang.RuntimeException (producer-properties valid-config2)))
      (is (thrown? java.lang.RuntimeException (producer-properties valid-config3))))

```
That would make it cumbersome, I believe. I feel, it's clear at the moment.

Reply:
yes something like this

Reply:
Keeping it how it is as of now.